### PR TITLE
Super Cache: Overhaul styling to match Jetpack aesthetics

### PR DESCRIPTION
**Pull request by jsnmoon**
Retry of #26890. cc @mikestottuk 

## Proposed changes:

After Easy 2

* Adds a new `dashboard.css` which introduces styling for:
    * A new header, footer, navigation component, and card component.
* Applies these new styles to all known Super Cache pages:
    * Easy, Advanced, CDN, Contents, Preload, Plugins, and Debug pages.

## Screenshots

Page|Before|After
-|-|-
Easy|Before Easy|After Easy
Advanced|Before Advanced|After Advanced
CDN|Before CDN|After CDN
Contents|Before Contents|After Contents
Preload|Before Preload|After Preload
Plugins|Before Plugins|After Plugins
Debug|Before Debug|After Debug

## Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

Prior discussion: pc9hqz-1e4-p2

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* Apply these changes to your test site.
* Ensure that the new styling has been applied to each Super Cache settings tab.

### CODE REVIEW

**Review by dilirity — COMMENTED** [review]
Got a warning right after I deployed a JN for testing.

> Warning: filemtime(): stat failed for styling/dashboard.css in /srv/users/user9cf56684/apps/user9cf56684/public/wp-content/plugins/wp-super-cache-dev/wp-cache.php on line 137

**Review by dilirity — COMMENTED** [review]
Wow - this looks great!

I've noticed somethings that seem off. Since I'm not sure if this is going to have a part 2 (there are a few TODOs in there), I'll leave them below.

### The footer seems off compared to Boost

**Super Cache:**
image

**Boost:**
CleanShot 2023-09-12 at 11 31 59

### Text readability

I can't put my finger on it, but the text throughout the UI seems off to me. It's a bit squished together. For example this feels noisy:

CleanShot 2023-09-12 at 11 34 18

### Boost banner

The banner spacing on the right also seems off:
CleanShot 2023-09-12 at 11 35 00

Just checked the original - on wider screens, spacing on the right is okay. On smaller it isn't.

### Debug tab

The sidebar is squished
CleanShot 2023-09-12 at 11 46 35

**Comment by oskosk** [comment]
I'm getting `Warning: filemtime(): stat failed for styling/dashboard.css in /wp-content/plugins/wp-super-cache-dev/wp-cache.php on line 137` when on wp-admin (any page)

**Review by donnchawp — APPROVED** [review]
Looks great! A much improved look to the settings pages.

**Review by donnchawp — DISMISSED** [review]
Oops, I didn't notice the warning about filemtime() on line 137 of wp-cache.php that Oscar saw. Forgot to tail php_errors. Should be an easy fix.
Other than that, it looks great.

**Comment by jsnmoon** [comment]
Good catch with the PHP warning; the incorrect invocation has been fixed. I also adjusted the line height issue Peter pointed out, but we can punt the rest of the refinements for later and hopefully smaller PR.

File: projects/plugins/super-cache/wp-cache.php [hunk 1]
```diff
@@ -133,6 +133,15 @@ function wp_cache_set_home() {
...
 
+function wpsc_enqueue_styles() {
+	wp_enqueue_style(
+		'wpsc_styles',
+		plugins_url( 'styling/dashboard.css', __FILE__ ),
+		array(),
+		filemtime( plugin_dir_path( __FILE__ ) . 'styling/dashboard.css' )
+	);
+}
+add_action( 'admin_enqueue_scripts', 'wpsc_enqueue_styles' );
```
Comment (donnchawp) [inline projects/plugins/super-cache/wp-cache.php]: The styles are leaking into wp-admin. Checking for $_GET['page'] === 'wpsupercache' before adding the action should fix it.

**Review by donnchawp — CHANGES_REQUESTED** [review]
Looks great, but the styles are leaking into the rest of wp-admin.

File: projects/plugins/super-cache/wp-cache.php (hunk 1 above)
Comment (jsnmoon) [reply projects/plugins/super-cache/wp-cache.php]: Fixed in https://github.com/Automattic/jetpack/pull/32981/commits/ca3d9f92c3bdf7b2b70911b4bc9f38902bd85b9b 👍

**Review by donnchawp — DISMISSED** [review]
Looks great! Works as advertised! Thanks for this.

**Review by mdbitz — APPROVED** [review]
Verified previous approval was dismissed due to merge. Re-approved.

**Comment by thingalon** [comment]
Very nice @jsnmoon 👍

**Pull request merged**